engine/schema/src/main/java/com/cloud/storage/VolumeGroupVO.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 The Apache Software Foundation.
```

Comment:
@DK101010, based in the documentation regarding Apache's source code, we should not add the copyright line to the header (see https://github.com/apache/cloudstack/pull/4954#discussion_r625084415). Could you take a look on all the files you added, please?